realm/realm-library/build.gradle

```diff
@@ -92,6 +92,9 @@ android {
     packagingOptions {
         exclude 'META-INF/NOTICE.txt'
         exclude 'META-INF/LICENSE.txt'
+        // We did strip with cmake for release build.
+        // Please, Gradle, you are not that smart! Pleas DO NOT strip debug symbols for debug build!
+        doNotStrip "*/*/*.so"
```

Comment:
after cmake, the release lib has been already stripped. We did stripping for release even before Android Plug has this smart move. https://github.com/realm/realm-java/blob/master/realm/realm-library/src/main/cpp/CMakeLists.txt#L207